uint32 pedID
uint32 frameID
float64 x
float64 y

#undecided data type
